### PR TITLE
Pagina de registros creada y llave primaria agregada a Desarrolla

### LINKED ISSUE #14: Crear pagina de registro
Solo poner codigo HTML y css
nada funcional

### LINKED ISSUE #28: Arreglar Desarrolla Seeder
añadir columna id a todas las tablas

### PR DESCRIPTION
**Pull request by AlainEd**
Resolve #14 

Resolve #28 

Pagina de registros creada, y llave primaria agregada a la tabla desarrolla.

### CODE REVIEW

File: database/migrations/2014_10_12_100000_create_password_resets_table.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('password_resets');
+    }
+}
```
Comment (JrogeT) [inline database/migrations/2014_10_12_100000_create_password_resets_table.php:32]: and this new file? :o

File: resources/views/master.blade.php [hunk 2]
```diff
@@ -10,11 +10,13 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
         integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="shortcut icon" href="{{ asset('favicon.ico') }}">
+    <link rel="stylesheet" href="{{asset('css/app.css')}}">   <!-- BOOTSTRAP -->
```
Comment (JrogeT) [inline resources/views/master.blade.php:13]: we are using bootstrap in two ways:
- CND (link)
- Downloaded (asset)

Use only one

File: routes/web.php [hunk 3]
```diff
@@ -17,8 +18,13 @@
 
 Route::get('/', loginController::class)->name('login');
 
+//ruta por demas, la de arriba llama al login como ruta principal
 Route::get('/login', function () {
     return view('login');
 });
+//---------------------------------------------------------------
```
Comment (JrogeT) [inline routes/web.php:25]: You're right, remove this route pls

**Review by JrogeT — CHANGES_REQUESTED** [review]

File: database/migrations/2014_10_12_100000_create_password_resets_table.php (hunk 1 above)
Comment (AlainEd) [reply database/migrations/2014_10_12_100000_create_password_resets_table.php:32]: ammmmm instalacion de bootstrap con --auth xD

**Pull request closed without merging**